pkg/app/app_reconcile.go

```diff
@@ -80,8 +80,11 @@ func (a *App) reconcileDeploy() error {
 	result := a.reconcileFetchTemplateDeploy()
 	a.setReconcileCompleted(result)
 
-	// Reconcile inspect regardless of deploy success
-	_ = a.reconcileInspect()
+	deployAttempted := a.Status().Deploy == nil
+	// Reconcile inspect regardless of deploy success but don't inspect if deploy never attempted
+	if !deployAttempted {
```

Comment:
other methods in this class use `a.app.Status...` so for consistency we should switch to `a.app.Status.Deploy`.

`a.app.Status.Deploy != nil` is descriptive enough imho for the conditional so var can be inlined.

Reply:
Addressed